src/lib/format/main/index.ts

```diff
@@ -1,24 +1,22 @@
 import fs from 'node:fs';
+import { parse as parseVue } from '@vue/compiler-sfc';
 
 import tmp from 'tmp';
 
 import { logger } from '../../common';
 import { type Configuration } from '../../config';
-import {
-  enhanceConfig,
-  type FormatOptions,
-} from '../config';
+import { enhanceConfig, type FormatOptions } from '../config';
 import { formatSource } from './format';
 
 /**
  * A type representing file extensions supported.
  */
-export type Extension = 'js' | 'ts' | 'jsx' | 'tsx';
+export type Extension = 'js' | 'ts' | 'jsx' | 'tsx' | 'vue';
```

Comment:
I didn't test but I doubt Vue is supported out of the box.

Are there more changes expected? If yes, you can bundle them in the next PR.

Reply:
Maybe need to add some tests more for publish a new version